transitgateway.tf

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------------------------------------------------
+# Associate to the TGW VPN attachment a non-default route table that
```

Comment:
```suggestion
# Associate the TGW VPN attachment to a non-default route table that
```

Or possibly "with" - the current phrasing is just a bit harder to parse than the suggestion.